apps/health_campaign_field_worker_app/lib/main.dart

```diff
@@ -25,8 +33,33 @@ class MainApplication extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    Client client = Client();
+
     return MultiBlocProvider(
       providers: [
+        BlocProvider(
+          create: (context) => AppInitilizationBloc(
+            const AppInitilizationState(),
+            MdmsRepository(client.init()),
+          )..add(const AppInitilizationSetupEvent()),
+          lazy: false,
+        ),
+        BlocProvider(
+          create: (context) => AuthBloc(
+            const AuthState(),
+          ),
+        ),
+        BlocProvider(
+          create: (context) => LocalizationBloc(
+            const LocalizationState(),
+            LocalizationRepository(client.init()),
+          )..add(const LocalizationEvent.onLoadLocalization(
+              module: 'hcm-common',
+              tenantId: 'pb',
```

Comment:
let's move the module, tenant and locale to some app config so it isn't hard-coded.

Reply:
@egov-joy  updated